_data/locales/en.yml

```diff
@@ -7,20 +7,23 @@ en:
     question: What Does Homebrew Do?
     what: Homebrew installs <a href="https://formulae.brew.sh/formula/" title="List of Homebrew packages">the stuff you need</a> that Apple (or your Linux system) didn’t.
     how: Homebrew installs packages to their own directory and then symlinks their files into <code>/opt/homebrew</code> (on Apple Silicon).
-    prefix: Homebrew won’t install files outside its prefix and you can place a Homebrew installation wherever you like.
-    createpackages: Trivially create your own Homebrew packages.
-    hack: It’s all Git and Ruby underneath, so hack away with the knowledge that you can easily revert your modifications and merge upstream updates.
+    prefix: Homebrew won’t install files outside its prefix, but you can place a Homebrew installation wherever you want.
+    createpackages: You can trivially create your own Homebrew packages.
+    hack: After all, it’s just Git and Ruby underneath, you can easily hack away with the knowledge, revert your modifications, and merge upstream updates.
+    submit: And if you want, you can submit your package to us <a href="https://github.com/Homebrew/homebrew-core/pulls">here</a>!
```

Comment:
This should link to the docs because casks exist too

Reply:
Yeah true, I'll get to doing that.